src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -117,12 +134,70 @@ const getAccountNumberSchema = (
   }
 }
 
+const getIbanNumberSchema = (
+  implicitParams: {
+    country: string
+    fiatAccountType: FiatAccountType
+  },
+  countryOverrides?: FiatAccountSchemaCountryOverrides
+): FiatAccountFormSchema<FiatAccountSchema.IBANNumber> => {
+  const overrides = countryOverrides?.[implicitParams.country]?.[FiatAccountSchema.IBANNumber]
+  //Ensure spaces/lower case don't lead to invalid input
+  const cleanInput = (input: string) => input.replace(/\s/g, '').toUpperCase()
+
+  const ibanNumberValidator = (input: string) => {
+    input = cleanInput(input)
+
+    if (overrides?.iban?.regex) {
+      const isValid = new RegExp(overrides.iban.regex).test(input)
+      return {
+        isValid,
+        errorMessage: i18n.t(`fiatAccountSchema.ibanNumber.errorMessage`),
+      }
+    }
+
+    const { valid, errorCodes } = validateIBAN(input)
+    if (!valid && input.length) {
```

Comment:
not sure the `&& input.length` part makes sense to me. We don't do that for the regex validation functions. As written, it seems like you could have an invalid IBAN with no error message. Is there a reason we'd want that?